### ISSUE TITLE
[Feature request] : duplicate and move entries

### ISSUE THREAD

**Issue by ricounet67** [buttercup/buttercup-desktop#1047]
Do you know if it possible to remove duplicate entries (coming for example from different importations) and is it possible to make a multiple selection to move several entries from a folder to another ?

Thank you

**Comment by perry-mitchell** [comment]
Moving entries is already possible in the UI. I'll mark this as a feature request for the cloning, however..

**Comment by ricounet67** [comment]
Yes, but it is very slow, and select multiple entries to move them in another folder is impossible
